sdk/operationalinsights/arm-operationalinsights/CHANGELOG.md

```diff
@@ -1,15 +1,150 @@
+  - Interface Workspace has a new optional parameter failover
+  - Interface Workspace has a new optional parameter replication
+  - Interface WorkspaceFeatures has a new optional parameter associations
+  - Interface WorkspaceFeatures has a new optional parameter unifiedSentinelBillingOnly
+  - Interface WorkspacePatch has a new optional parameter failover
+  - Interface WorkspacePatch has a new optional parameter replication
+  - Added Type Alias AccessRuleDirection
+  - Added Type Alias ClusterReplicationState
+  - Added Type Alias IssueType
+  - Added Type Alias ManagedServiceIdentityType
+  - Added Type Alias NetworkSecurityPerimeterConfigurationProvisioningState
+  - Added Type Alias ResourceAssociationAccessMode
+  - Added Type Alias RuleTypeEnum
+  - Added Type Alias Severity
+  - Added Type Alias StatusCodeEnum
+  - Added Type Alias SummaryLogsCreateOrUpdateResponse
+  - Added Type Alias SummaryLogsDeleteResponse
+  - Added Type Alias SummaryLogsGetResponse
+  - Added Type Alias SummaryLogsListByWorkspaceNextResponse
+  - Added Type Alias SummaryLogsListByWorkspaceResponse
+  - Added Type Alias SummaryLogsRetryBinResponse
+  - Added Type Alias SummaryLogsStartResponse
+  - Added Type Alias TimeSelectorEnum
+  - Added Type Alias WorkspaceFailoverState
+  - Added Type Alias WorkspaceReplicationState
+  - Added Type Alias WorkspacesFailbackResponse
+  - Added Type Alias WorkspacesFailoverResponse
+  - Added Type Alias WorkspacesGetNSPResponse
+  - Added Type Alias WorkspacesListNSPNextResponse
+  - Added Type Alias WorkspacesListNSPResponse
+  - Added Type Alias WorkspacesReconcileNSPResponse
+  - Added Enum KnownAccessRuleDirection
+  - Added Enum KnownClusterReplicationState
+  - Added Enum KnownIssueType
+  - Added Enum KnownManagedServiceIdentityType
+  - Added Enum KnownNetworkSecurityPerimeterConfigurationProvisioningState
+  - Added Enum KnownResourceAssociationAccessMode
+  - Added Enum KnownRuleTypeEnum
+  - Added Enum KnownSeverity
+  - Added Enum KnownStatusCodeEnum
+  - Added Enum KnownTimeSelectorEnum
+  - Added Enum KnownWorkspaceFailoverState
+  - Added Enum KnownWorkspaceReplicationState
+  - Enum KnownProvisioningStateEnum has a new value Deleting
+  - Enum KnownPublicNetworkAccessType has a new value SecuredByPerimeter
+  - Enum KnownTablePlanEnum has a new value Auxiliary
 
 ### Breaking Changes
+  - Operation Clusters.beginCreateOrUpdate has a new signature
+  - Operation Clusters.beginCreateOrUpdateAndWait has a new signature
+  - Operation Clusters.beginUpdate has a new signature
+  - Operation Clusters.beginUpdateAndWait has a new signature
+  - Operation Clusters.get has a new signature
+  - Operation Workspaces.beginCreateOrUpdate has a new signature
+  - Operation Workspaces.beginCreateOrUpdateAndWait has a new signature
+  - Operation Workspaces.get has a new signature
+  - Operation Workspaces.update has a new signature
```

Comment:
do you know what changes for these two operations?

```
  - Operation Workspaces.get has a new signature
  - Operation Workspaces.update has a new signature
```

Reply:
this is because the response model has diff
<img width="919" height="693" alt="image" src="https://github.com/user-attachments/assets/7ab4ef39-ce7a-4c1e-9e6e-13e8524a0ba5" />

Reply:
Could you create an issue for this and it is wired?

Reply:
https://github.com/Azure/azure-sdk-tools/issues/13431